### DISCUSSION TITLE
Preserve exif when using compose ?

### DISCUSSION THREAD

**Discussion by miharix** [ImageMagick/ImageMagick#7499] [Help] [answered]
### ImageMagick version

7.1.1-33 Q16-HDRI x86_64 22263

### Operating system, version and so on

Manjaro 24.0.3

### Description

Hi 
I'm trying to batch mask multiple images and it works great using 

`magick input.tif mask.tif -alpha off -compose copy_opacity -composite output.tif `

but unfortunately all exif data gets lost 

I also tried
`magick -define tiff:exif-properties=true input.tif mask.tif -alpha off -compose copy_opacity -composite output.tif `
but It's no difference.

Can someone please point me to the correct parameter ?

Tnx!

My test tif files are here
https://we.tl/t-xi9M7O559y

**Comment by urban-warrior** [comment, marked as answer]
ImageMagick does not support writing the EXIF profile to TIFF images.
